src/axolotl/utils/schemas/peft.py

```diff
@@ -128,6 +128,16 @@ class LoraConfig(BaseModel):
             "description": "loraplus learning rate for lora embedding layers. Default value is 1e-6."
         },
     )
+    use_qalora: bool | None = Field(
+        default=False,
+        json_schema_extra={
+            "description": "Enable Quantization-Aware Low-Rank Adaptation"
+        },
+    )
```

Comment:
```suggestion
```
This feels unneeded as we can specify via `adapter: use_qalora`. I think we can remove other places this is referenced too.